src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/NotifyConfiguration.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import uk.gov.service.notify.NotificationClient
+
+@Configuration
+class NotifyConfiguration(@Value("\${notify.api.key:invalidKey}") private val apiKey: String) {
+  @Bean
+  fun notifyClient(): NotificationClient {
```

Comment:
How does the client communicate with the API? I don't see a URL anywhere

Reply:
Think this is built into the java client as a default value. It can take a proxy, but I don't think we need one. e.g. `NotificationClient client = new NotificationClient(apiKey, proxy);`